[core]/es_extended/locales/da.lua

```diff
@@ -0,0 +1,366 @@
+Locales['en'] = {
```

Comment:
```suggestion
Locales['da'] = {
```

Reply:
since it is Danish and not English I think it is correct "da " and not "en"

Reply:
> since it is Danish and not English I think it is correct "da " and not "en"

oh shet forgot to edit that part